imagetest/fixtures.go

```diff
@@ -58,6 +58,19 @@ type TestVM struct {
 	instance     *daisy.Instance
 }
 
+// TestVMParams stores config params for a test vm
+type TestVMParams struct {
+	Disks []*compute.Disk
+	// this bool pointer must be explicitly initialized to use
+	// the special guest attribute for tests which reboot
+	VMRebootsDuringTest *bool
```

Comment:
Maybe this should extend the existing `func (t *testVM) Reboot` instead of being a separate parameter, or be a separate `RebootViaManager` function that explains the difference. That would make everything else cleanly map to an option that can be set directly on a compute.Instance or a daisy.Instance

Reply:
The planned solution in a new pr is to set the *daisy.Instance metadata map directly